File: frontend/projects/api/src/lib/evidence-metadata.ts
import type { SourceCitation } from './source-citation';

export interface EvidenceMetadata {
  confidence: 'high' | 'medium' | 'low' | 'unknown';
  citations: SourceCitation[];
  assumptions: string[];
  omissions: string[];
  conflictingSources: string[];
  unresolvedQuestions: string[];
}

File: backend/src/RepoFluent.Application/EvidenceMetadata.cs
namespace RepoFluent.Application;

public sealed record EvidenceMetadata(
    string Confidence,
    IReadOnlyList<SourceCitation>? Citations,
    IReadOnlyList<string>? Assumptions,
    IReadOnlyList<string>? Omissions,
    IReadOnlyList<string>? ConflictingSources,
    IReadOnlyList<string>? UnresolvedQuestions);
